### ISSUE TITLE
Incompatibility with Windows 7

### ISSUE THREAD

**Issue by schlaus** [mohebifar/racket#11]
The command `npm run build` and thus `yo racket` as well fails on Win 7.

Here's the error:

```
C:\Users\Schlaus\Documents\racket\node_modules\.bin\webpack:2
basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
          ^^^^^^^

SyntaxError: missing ) after argument list
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Function.Module.runMain (module.js:441:10)
    at startup (node.js:139:18)
    at node.js:968:3
```

Looking at package.json it seems the `C:\Users\Schlaus\Documents\racket\node_modules\.bin\webpack` is run via node:

package.json, line 13
`"build": "node ./node_modules/.bin/webpack --colors --config ./webpack/prod.config.js"`

Is this intentional? Removing `node ` from the start of the line makes the command work.

Besides that, scripts in package.json that use the syntax

`NODE_ENV=production NODE_PATH=./src node ./bin/server.js`

ie. define a variable at the start wont work on Win 7 without the cross-env package:

`cross-env NODE_ENV=production NODE_PATH=./src node ./bin/server.js`

Could you consider adding cross-env, or mention in the docs which steps need to be taken for Win 7 compatibility?

**Comment by mohebifar** [comment]
Thanks for the report. I will fix this with `better-npm-run`.

**Comment by mohebifar** [comment]
Fixed in 88d3290.
